packages/knowledge/src/base.ts

```diff
@@ -450,7 +450,7 @@ export const base: ChartKnowledgeBase = {
       { minQty: 1, maxQty: 1, fieldConditions: ['Nominal', 'Ordinal'] },
       { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] },
     ],
-    channel: ['Angle', 'Area', 'Color'],
+    channel: ['Angle', 'Color'],
```

Comment:
I would say Area is one of the channels of pie charts. There are papers discussed this, I believe.

Reply:
users will see them in diff ways.

e.g. pie charts have 'angles' while donut charts have not, but both of them have 'area'. Its the way that an average user can see.

Reply:
emmm